test/plugins/metrics/metrics_suite_test.go

```diff
@@ -4,12 +4,20 @@ import (
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/types"
 	. "github.com/onsi/gomega"
 	_ "github.com/rancher/opni/pkg/test/setup"
 	_ "github.com/rancher/opni/plugins/metrics/test"
 )
 
 func TestMetrics(t *testing.T) {
+	suiteConfig := types.NewDefaultSuiteConfig()
```

Comment:
did you mean to leave these in here?

Reply:
nope, I'll take those out thanks for catching